### PR TITLE
added version and start.sh files

### PR DESCRIPTION
**Pull request by ramkch**

### CODE REVIEW

**Review by dominicst-ch — APPROVED** [review]

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# Start script for insolvency-delta-consumer
+
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [[ -z "${MESOS_SLAVE_PID}" ]]; then
+    source ~/.chs_env/private_env
+    source ~/.chs_env/global_env
+    source ~/.chs_env/company-profile-api/env
+
+    PORT="${OFFICER_DELTA_PROCESSOR_PORT}"
```
Comment (dominicst-ch) [inline start.sh:12]: Just realised that this env var is specifically for the officers processor - we should rename

Comment (dominicst-ch) [reply start.sh:12]: May need to go back and ammend other start.sh files

**Review by dominicst-ch — CHANGES_REQUESTED** [review]

File: start.sh (hunk 1 above)
Comment (ramkch) [reply start.sh]: changed to {COMPANY_PROFILE_API_PORT} , presume that will be set as env var.

File: start.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Start script for insolvency-delta-consumer
```
Comment (dominicst-ch) [inline start.sh:3]: This says `insolvency-delta-consumer` still

**Review by dominicst-ch — CHANGES_REQUESTED** [review]

**Review by dominicst-ch — APPROVED** [review]

**Pull request merged**